CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contributing to Nick Project
+
+Thank you for your interest in contributing to Nick Project! This document outlines the guidelines for contributing to this project.
+
+## Table of Contents
+
+- Contributing to Nick Project
+  - [Table of Contents](#table-of-contents)
+  - [Branching Rules](#branching-rules)
+    - [Types of Branches](#types-of-branches)
+    - [Branch Naming Conventions](#branch-naming-conventions)
+      - [Examples](#examples)
+  - [Commit Convention](#commit-convention)
+  - [Pull Requests](#pull-requests)
+    - [For Team Members:](#for-team-members)
+    - [For External Contributors:](#for-external-contributors)
+
+## Branching Rules
+
+### Types of Branches
+
+- `main`: For production use. Only accepts PRs from `develop` (and `hotfix` in emergencies).
+- `feature/`: Branches for developing new features or improvements.
+- `hotfix/`: Branches for urgent fixes.
+- `docs/`: Branches for documentation updates.
+
+### Branch Naming Conventions
+
+- Feature Addition: `feature/<issue_number>-<brief_description>` (develop ⇔ feature)
+- Hotfix: `hotfix/<issue_number>-<brief_description>` (main ⇔ hotfix)
+- Documentation Update: `docs/<issue_number>-<brief_description>` (develop ⇔ docs)
+
+*Note: `<issue_number>-` can be omitted if the modification is not based on an issue.*
+
+
+#### Examples
+
+- feature/42-user-authentication
+- hotfix/56-security-vulnerability-fix
+- docs/108-update-installation-guide
+
+## Commit Convention
+
+Before you create a Pull Request, please check whether your commits comply with
+the commit conventions used in this repository.
+
+When you create a commit we kindly ask you to follow the convention
+`category(scope or module): message` in your commit message while using one of
+the following categories:
+
```

Comment:
@hitohata 
Could you please replace this part with the following part? Three prefixes are added.

```md
- `build`: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
- `ci`: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
- `docs`: Documentation only changes
- `feat`: A new feature
- `fix`: A bug fix
- `perf`: A code change that improves performance
- `refactor`: A code change that neither fixes a bug nor adds a feature
- `revert`: Reverts a previous commit
- `style`: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
- `test`: Adding missing tests or correcting existing tests

If you are interested in the detailed specification you can visit:

- [Conventional Commits](https://www.conventionalcommits.org/)
- [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#commitlintconfig-conventional)
- [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines).
```

Reply:
I did it, Sir.

Reply:
Thank you!!
Don't call me sir 😅